src/main/resources/io/jenkins/plugins/pipeline_elasticsearch_logs/ElasticSearchConfiguration/help-certificateId.html

```diff
@@ -1,4 +1,5 @@
 <div>
-  A SSL certificate used to validate the certificate of the Elasticsearch server when using https.<br/>
-  In case you have a certificate in the truststore of your JVM running Jenkins you can leave this empty.
+  A truststore with a SSL certificate used to validate the certificate of the Elasticsearch server when using https.<br/>
```

Comment:
This means the change is incompatible (in case someone would already use the plugin with a certificate)?
We should at least increase the minor version in this case.

Reply:
This change is only incompatible when you specified a certificate which is actually not used because the trust comes from the default JVM truststore.
Regarding the version we should anyway align the version with the releases in github